src/DialogMessage.ts

```diff
@@ -8,29 +8,23 @@ export namespace DialogMessage {
     export const no: MessageItem = { title: localize('tomcatExt.no', 'No'), isCloseAffordance: true };
     export const cancel: MessageItem = { title: localize('tomcatExt.cancel', 'Cancel'), isCloseAffordance: true };
     export const never: MessageItem = { title: localize('tomcatExt.never', 'Never') };
-    export const moreInfo: MessageItem = { title: localize('tomcatExt.more', 'More Info') };
+    export const moreInfo: MessageItem = { title: localize('tomcatExt.moreInfo', 'More Info') };
     export const selectServer: string = localize('tomcatExt.selectServer', 'Select Tomcat Server');
     export const createServer: string = localize('tomcatExt.createServer', 'Create New Server');
     export const noServer: string = localize('tomcatExt.noServer', 'There are no Tomcat Servers.');
-    export const serverExist: string = localize('tomcatExt.serverExist', 'This Tomcat Server already exists, abort creation.');
     export const noPackage: string = localize('tomcatExt.noPackage', 'The selected package is not under current workspace.');
     export const noServerConfig: string = localize('tomcatExt.noServerConfig', 'The Tomcat Server is broken. It does not have server.xml');
     export const selectWarPackage: string = localize('tomcatExt.selectWarPackage', 'Select War Package');
     export const selectDirectory: string = localize('tomcatExt.selectDirectory', 'Select Tomcat Directory');
     export const deleteConfirm: string = localize('tomcatExt.deleteConfirm', 'This Tomcat Server is running, are you sure you want to delete it?');
     export const serverRunning: string = localize('tomcatExt.serverRunning', 'This Tomcat Server is already started.');
     export const serverStopped: string = localize('tomcatExt.serverStopped', 'This Tomcat Server was stopped.');
-    export const continueOnExistingServer: string = localize('tomcatExt.continueOnExistingServer', 'This Tomcat Server already exists. Do you want to continue the operation on this server?');
-    export const startServerToBrowseWarPackage: string = localize('tomcatExt.startServerToBrowseWarPackage', 'Start Tomcat Server to browse the war package');
+    export const startServer: string = localize('tomcatExt.startServer', 'Would you like to start Tomcat Server first to browse the war package?');
```

Comment:
If the user cannot browse the war package without starting the server, then this should be phrased like:
The Tomcat server needs to be started before browsing the war packages. Would you like to start it now?

Reply:
And the actions are: Yes, No.

Reply:
what a good prompt message! Actions are yes & no.